examples/nft/scripts/test.sh

```
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+function nft_balance() {
+  echo -e "\n🖼️  NFT Balance"
+  echo "---------------------------------------------"
+  local ZETACHAIN=$(cast call "$CONTRACT_ZETACHAIN" "balanceOf(address)(uint256)" "$SENDER")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)